ipn/localapi/localapi.go

```diff
@@ -2626,3 +2627,21 @@ var (
 	// User-visible LocalAPI endpoints.
 	metricFilePutCalls = clientmetric.NewCounter("localapi_file_put")
 )
+
+// serveSuggestDerpExitNode serves a POST endpoint for returning a suggested exit node.
+func (h *Handler) serveSuggestDERPExitNode(w http.ResponseWriter, r *http.Request) {
+	if !h.PermitWrite {
+		http.Error(w, "access denied", http.StatusForbidden)
+		return
+	}
```

Comment:
It's not a write anymore, this could be relaxed maybe. But if it's going to trigger ICMP pings in future perhaps we should leave this -- those aren't a write, but it's also a fairly invasive operation.